chainer/functions/loss/negative_sampling.py

```diff
@@ -359,22 +364,37 @@ def negative_sampling(x, t, W, sampler, sample_size, reduce='sum'):
         sample_size (int): Number of samples.
         reduce (str): Reduction option. Its value must be either
             ``'sum'`` or ``'no'``. Otherwise, :class:`ValueError` is raised.
+        return_samples (bool):
+            If ``True``, the sample array taken by ``sampler`` is also
+            returned.
 
     Returns:
-        ~chainer.Variable:
-            A variable holding the loss value(s) calculated by the
-            above equation.
+        ~chainer.Variable or tuple:
+            If ``return_samples`` is ``False`` (default), the output
+            variable holding the loss value(s) calculated by the
+            above equation is returned.
             If ``reduce`` is ``'no'``, the output variable holds array
             whose shape is same as one of (hence both of) input variables.
             If it is ``'sum'``, the output variable holds a scalar value.
 
+            If ``return_samples`` is ``True``, a tuple of the output
```

Comment:
How about putting this sentence right after the "False" case description, above the description of "reduce" option? I was first confused by the combinations of interleaved "if" sentences.

Reply:
Fixed.